app/models/phase.rb

```diff
@@ -122,6 +41,7 @@ def deep_copy(**options)
     return copy
   end
 
+# TODO: Move this to Plan model
```

Comment:
Agreed that we would need to update the view to work with the change. I was thinking we could change the method to `num_answered_questions(phase=nil)` and either return the # of answered questions for the specified phase or the whole plan if none is specified. I think it makes more sense for it to live on the plan model since we'd never want to know how many answers the phase has across all plans. The plan is the limiting factor.

Reply:
More than answers across all plans is that from a phase perspective there is no concept of answer at all, similar token applies for section but right now is not an issue in the UI,  is more a problem of where these methods should be placed instead and perhaps at the plan is the best candidate since that model knows what is phase, section and answer...